azure-pipelines.yml

```diff
@@ -1,22 +1,36 @@
 # Maven
 
 trigger:
-- master
+  - master
 
 pool:
   vmImage: 'windows-latest'
 
 steps:
-- task: ScreenResolutionUtility@1
-  inputs:
-    displaySettings: 'optimal'
-- task: Maven@3
-  inputs:
-    mavenPomFile: 'pom.xml'
-    mavenOptions: '-Xmx3072m'
-    javaHomeOption: 'JDKVersion'
-    jdkVersionOption: '1.8'
-    jdkArchitectureOption: 'x64'
-    publishJUnitResults: true
-    testResultsFiles: '**/surefire-reports/TEST-*.xml'
-    goals: 'clean test -DdriverSettings.chrome.webDriverVersion=75.0.3770.140 -Dprofile=local'
+  - task: SonarCloudPrepare@1
+    inputs:
+      SonarCloud: 'SonarCloud'
+      organization: 'aqualityautomation'
```

Comment:
ERROR: Error during SonarQube Scanner execution
 Check failure on line 187 in Build log

Build log #L187
ERROR: main component does not belong to specified organization
ERROR: 
ERROR: Re-run SonarQube Scanner using the -X switch to enable full debug logging.

Reply:
fixed